pkgs/applications/graphics/meshlab/default.nix

```diff
@@ -17,7 +17,7 @@
 
 mkDerivation rec {
   pname = "meshlab";
-  version = "2020.03";
+  version = "2020.07";
```

Comment:
Hold on - you've changed the `version`, but you haven't changed anything about the `fetchFromGitHub` call below. So it's still building the old code.

Reply:
OMG! Now it's fixed